static/local/blackwomenssuffrage/1820-1859.md

```diff
@@ -6,7 +6,7 @@
 
 **1843**: Isabella Baumfree takes the name [Sojourner Truth](/search?q=Sojourner+Truth) and becomes a traveling speaker, lecturing about the ills of slavery and, later, temperance and women's rights.
 
-**1848**: White suffragists gather at the Seneca Falls Convention. No Black women are invited to attend. Frederick Douglass, believed to be the only Black person in attendance, becomes the first man (black or white) to publicly call for women's suffrage. (Terborg-Penn, 14)
+**1848**: White suffragists gather at the Seneca Falls Convention. No Black women are invited to attend. Frederick Douglass, believed to be the only Black person in attendance, becomes the first man (Black or white) to publicly call for women's suffrage. (Terborg-Penn, 14)
```

Comment:
If we are capitalizing "Black" should we also capitalize "White"?

Reply:
Nope!